src/wallet/wallet2.cpp

```diff
@@ -13254,6 +13298,8 @@ size_t wallet2::import_outputs(const std::pair<uint64_t, std::vector<tools::wall
     td.m_key_image_known = etd.m_flags.m_key_image_known;
     td.m_key_image_request = etd.m_flags.m_key_image_request;
     td.m_key_image_partial = false;
+    td.m_subaddr_index.major = etd.m_subaddr_index_major;
+    td.m_subaddr_index.minor = etd.m_subaddr_index_minor;
```

Comment:
Small suggestion (perhaps for a future PR). Right before  `generate_key_image_helper` below:

 ```
 if (should_expand(td.m_subaddr_index))
     create_one_off_subaddress(td.m_subbaddr_index); 
 ```
 
This supports importing outputs sent to high index subaddresses without manual action needed by the user, and without calling the more expensive `expand_subaddresses` until the key image is confirmed generated properly.

Dev/user reported trying this [here](https://libera.monerologs.net/monero-dev/20220818#c137638)